buf.yaml

```diff
@@ -0,0 +1,17 @@
+version: v2
+modules:
+  - path: proto
+    name: buf.build/connectrpc/eliza
+lint:
+  use:
+    - DEFAULT
+  except:
```

Comment:
Can these exceptions be removed? IIRC the migrator adds these for backwards compat